apps/ledger-live-desktop/src/newArch/features/Collectibles/Ordinals/components/RareSats/TableHeader.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Text } from "@ledgerhq/react-ui";
+import { useTranslation } from "react-i18next";
+import RowLayout from "./RowLayout";
+
+export const TableHeader = () => {
+  const { t } = useTranslation();
+
+  const firstColumn = (
+    <Text variant="bodyLineHeight" fontSize={12} color="neutral.c70">
+      {t("ordinals.rareSats.table.type")}
+    </Text>
+  );
+
+  const secondColumn = (
+    <Text variant="bodyLineHeight" fontSize={12} color="neutral.c70">
+      {t("ordinals.rareSats.table.year")}
+    </Text>
+  );
+
+  const thirdColumn = (
+    <Text variant="bodyLineHeight" fontSize={12} color="neutral.c70">
+      {t("ordinals.rareSats.table.utxo")}
+    </Text>
+  );
```

Comment:
Not mandatory but you could have created a generic component that takes a string as parameter :)